<launch>
    <param name="use_sim_time" value="true" />

    <!--<include file="$(find localization)/launch/localization.launch"/>-->
    <include file="$(find localization)/launch/mocap_localization.launch"/>

    <node pkg="rosbag" type="play" name="player" output="screen" args="-r 1 --clock  -l /home/kostas/Dropbox/experiments/7/funny.bag"/>
    <!--<node pkg="rosbag" type="record" name="recorder" output="screen" args="-O /home/kostas/results/exp.bag /tf /poseupdate /odometry/wheel_imu /odometry/map /mocap_pose /tracks/mean /tracks/filtered /tracks/box /tracks/ukf /robot_1 /robot_2" />-->


    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find datmo)/config/datmo.rviz" />

   <node pkg="datmo" type="datmo" name="datmo" >
      <param name="write_execution_times" value="true" />
      <param name="lidar_frame" value="laser" />
      <param name="world_frame" value="map" />
      <param name="threshold_distance" value="0.2" />
      <param name="euclidean_distance" value="0.25" />
      <param name="pub_markers" value="true" />
   </node>

</launch>
